### PR TITLE
[TT-11985] DX-1287 MDCB 2.5.1 Config Changes

### PR DESCRIPTION
**Pull request by buger**
## **User description**
Triggered by: sredxny

Included:

Tyk Gateway: false
Tyk Dashboard: false
Tyk MDCB true
Tyk Pump false

Intended for: master
Changes sourced from: release-2.5
Config info generator branch: main

Note: <none> (branch suffix: docs)

JIRA: https://tyktech.atlassian.net/browse/TT-11985
https://tyktech.atlassian.net/browse/DX-1287

## Preview link:
https://deploy-preview-4506--tyk-docs.netlify.app/docs/nightly/tyk-multi-data-centre/mdcb-configuration-options/#http_server_options
___

## **Type**
Documentation, Enhancement

___

## **Description**
- Updated the `server_options` description to reflect gorpc server configuration.
- Introduced a comprehensive `http_server_options` section, including SSL/TLS settings, certificate management, and security protocols for the HTTP server.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>mdcb-config.md</strong><dd><code>Update MDCB Configuration Documentation and Add HTTP Server Options</code></dd></summary>
<hr>

tyk-docs/content/shared/mdcb-config.md
<li>Updated the description for <code>server_options</code> to specify it as gorpc <br>server configuration.<br> <li> Added a new section <code>http_server_options</code> detailing configuration for <br>SSL/TLS, certificate paths, TLS versions, cipher suites, and SSL <br>certificates.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4506/files#diff-4dc7d90d096907b83b780b481a0de5552fb107c44ec34d9e9f75099ae2ec77d4">+45/-1</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/shared/mdcb-config.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Type: `bool`<br />
 Enable debugging of your Tyk MDCB by exposing profiling information.
 
 ### server_options
-MDCB HTTP server configuration
+MDCB gorpc server configuration
```
Comment (dcs3spp) [inline tyk-docs/content/shared/mdcb-config.md:20]: @sredxny Why has HTTP server configuration being removed and replaced with only gorpc? Please could you arrange a peer review?

**Review by dcs3spp — APPROVED** [review]
PR LGTM thanks @sredxny

**Review by mativm02 — APPROVED** [review]

**Pull request merged**